tutorials/napps/development_environment_setup.rst

```diff
@@ -48,48 +48,10 @@ Install the dependencies necessary to build Python:
 
 .. code-block:: console
 
+ sudo add-apt-repository ppa:deadsnakes/ppa
  sudo apt update
- sudo apt install build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libssl-dev libreadline-dev libffi-dev libsqlite3-dev wget libbz2-dev
+ sudo apt install python3.9 
```

Comment:
@usmanazFIU did you have the chance to confirm if it also installs pip and the dev dependencies? I wonder if `python3.9-dev python3.9-venv` would also be needed.

Reply:
pip is installed with python installation. for the safe side we can add sudo apt install python3.9-dev and sudo apt install 
python3.9-venv

Reply:
from the internet 
![image](https://user-images.githubusercontent.com/104851066/175169544-6394f610-a515-4043-a8ee-1250182dad8c.png)